src/ert/analysis/_es_update.py

```diff
@@ -562,14 +563,24 @@ def analysis_IES(
     )
 
 
-def _write_update_report(fname: Path, snapshot: SmootherSnapshot) -> None:
-    # Make sure log file parents exist
+def _write_update_report(
+    path: Path, snapshot: SmootherSnapshot, run_id: str, global_scaling: float
+) -> None:
+    fname = path / f"{run_id}.txt"
     fname.parent.mkdir(parents=True, exist_ok=True)
     for update_step_name, update_step in snapshot.update_step_snapshots.items():
         with open(fname, "w", encoding="utf-8") as fout:
             fout.write("=" * 127 + "\n")
-            fout.write("Report step...: deprecated\n")
-            fout.write(f"Update step......: {update_step_name:<10}\n")
+            timestamp = datetime.now().strftime("%Y.%m.%d %H:%M:%S")
+            fout.write(f"Time: {timestamp}\n")
+            fout.write(f"Parent ensemble: {snapshot.source_case}\n")
+            fout.write(f"Target ensemble: {snapshot.target_case}\n")
+            fout.write(f"Alpha: {snapshot.alpha}\n")
+            fout.write(f"Global scaling: {global_scaling}\n")
+            fout.write(f"Standard cutoff: {snapshot.std_cutoff}\n")
+            fout.write(f"Run id: {run_id}\n")
+            fout.write("Report step: deprecated\n")
```

Comment:
Suggest just dropping this while you are at it, it has been deprecated for years.